content/en/blogs/serverless-vs-faas.md

```diff
@@ -0,0 +1,96 @@
+- **Less cost**: With FaaS, you only have to pay when the function is triggered and executed, without the need to spend any money on unnecessary cloud resources.
+
+### Disadvantages
+
+The disadvantages of FaaS mainly lie in:
+
+- **Limited functionality**: The functions in FaaS are small pieces of code used to accomplish simple tasks. If you want to write complex functions, FaaS may not be your best choice.
+- **Difficulty in function management**: Since you will be developing small pieces of function code, it could be difficult to manage a large number of functions as your business continues to grow.
+- **Data storage**: Functions in FaaS are mainly for running stateless workloads. How to find a way to store the data would be an issue if these functions have to call other stateful workloads.
+
+## Which One to Choose
+
+Serverless can be used in a wider range of functionalities, while FaaS focuses on providing an efficient way for the development of functions. As your business continues to grow, you may need to find yourself more tools to manage the functions, whereas this issue could be avoided if you choose other relatively complex serverless solutions with more components.
+
+Serverless offers more comprehensive solutions, while FaaS brings more flexibility. How to make the choice is always related to your practical needs. Moreover, it is not necessary to draw a line between these two. We can still decide to use them both if that suits our needs.
+
+### Serverless and FaaS providers
+
+There are a lot of serverless and FaaS providers available nowadays. For example:
+
+- AWS Lambda
+- Google Cloud Functions
+- Azure Functions
+
+Although many cloud providers have their serverless or FaaS solutions ready for developers, it is possible to face the issue of vendor lock-in after you run your business functions on a single platform.
+
+### Open-source serverless/FaaS alternatives
+
+In recent years, many outstanding open-source projects (such as [KEDA](https://keda.sh/), [Dapr](https://dapr.io/), [Cloud Native Buildpacks (CNB)](https://buildpacks.io/), [Tekton](https://tekton.dev/), and [Shipwright](https://shipwright.io/)) have emerged in the cloud-native Serverless-related fields. It is not easy to select the appropriate tools and put them together to build your own serverless solution.
+
+Thus, [OpenFunction](https://github.com/OpenFunction/OpenFunction/) comes as a cloud-native open-source FaaS platform aiming to let you focus on your business logic without having to maintain the underlying runtime environment and infrastructure. It can be easily deployed to your Kubernetes cluster. You can generate event-driven and dynamically scaling serverless workloads by simply submitting business-related source code in the form of functions.
```

Comment:
We can recommend readers here to learn more about the use case of OpenFunction through https://kubesphere.io/blogs/faas-openfunction/.

Also please add a hyperlink to 'FaaS platform' with https://openfunction.dev/

Reply:
Thanks for your ideas. Updated.